hammer/clients.go

```diff
@@ -127,6 +136,22 @@ func (r *LeafReader) Kill() {
 	}
 }
 
+// tileCache stores the results of the last fetched tile. This allows
+// readers that read contiguous blocks of leaves to act more like real
+// clients and fetch a tile of 256 leaves once, instead of 256 times.
+type tileCache struct {
```

Comment:
How about `leafBundleCache`? It took me a while to realise this was about leaf bundles rather than tree tiles.

Reply:
Sure.